slides/quickstart/index.html

```diff
@@ -24,53 +24,43 @@
     <p>Google Slides API Quickstart</p>
 
     <!--Add buttons to initiate auth sequence and sign out-->
-    <button id="authorize_button" style="display: none;">Authorize</button>
-    <button id="signout_button" style="display: none;">Sign Out</button>
+    <button id="authorize-button" style="display: none;">Authorize</button>
+    <button id="signout-button" style="display: none;">Sign Out</button>
+    <pre id="content"></pre>
 
-    <pre id="content" style="white-space: pre-wrap;"></pre>
-
-    <script type="text/javascript">
+    <script>
       // Client ID and API key from the Developer Console
-      var CLIENT_ID = '<YOUR_CLIENT_ID>';
-      var API_KEY = '<YOUR_API_KEY>';
-
+      const CLIENT_ID = '<YOUR_CLIENT_ID>';
+      const API_KEY = '<YOUR_API_KEY>';
       // Array of API discovery doc URLs for APIs used by the quickstart
-      var DISCOVERY_DOCS = ["https://slides.googleapis.com/$discovery/rest?version=v1"];
-
+      const DISCOVERY_DOCS = ["https://slides.googleapis.com/$discovery/rest?version=v1"];
       // Authorization scopes required by the API; multiple scopes can be
       // included, separated by spaces.
-      var SCOPES = "https://www.googleapis.com/auth/presentations.readonly";
-
-      var authorizeButton = document.getElementById('authorize_button');
-      var signoutButton = document.getElementById('signout_button');
+      const SCOPES = "https://www.googleapis.com/auth/presentations.readonly";
 
+      const AUTHORIZE_BUTTON = document.getElementById('authorize-button');
```

Comment:
I was wrong about using the upper underscore case here, you can do back to lower camel.